yarn.lock

```diff
@@ -5661,10 +5661,17 @@ through@2, "through@>=2.2.7 <3":
   resolved "https://registry.yarnpkg.com/through/-/through-2.3.8.tgz#0dd4c9ffaabc357960b1b724115d7e0e86a2e1f5"
   integrity sha1-DdTJ/6q8NXlgsbckEV1+Doai4fU=
 
+<<<<<<< HEAD
```

Comment:
a merge conflict here which should be addressed

Reply:
huh, so I have to actually save my files before comitting them.. gotcha.. ;)